packages/lib-classifier/src/plugins/drawingTools/experimental/models/marks/TranscriptionLine/TranscriptionLine.js

```diff
@@ -0,0 +1,79 @@
+import { types } from 'mobx-state-tree'
+import { TranscriptionLine as TranscriptionLineComponent } from '../../../components/'
+
+import { Mark } from '@plugins/drawingTools/models/marks'
+
+const TranscriptionLineModel = types
+  .model('TranscriptionLineModel', {
+    x1: types.maybe(types.number),
+    y1: types.maybe(types.number),
+    x2: types.maybe(types.number),
+    y2: types.maybe(types.number),
+    closed: types.optional(types.boolean, false)
```

Comment:
Is there a reason to use `closed` instead of PFE's `_inProgress`? This reverses the logic, so if the logic does have a reason to be reversed, then I might prefer `finished` to be consistent with the other action and function names being used around the code since closed === finished.

Reply:
`finished` sounds better to me. `closed` was copied from the polygon code, and is used by ASM (https://github.com/zooniverse/front-end-monorepo/issues/1314#issuecomment-564084683) but I don't think it's needed either.

I think that makes sense for polygons too, because my understanding is a polyline can be finished *and* either open or closed (I could be wrong.)

Reply:
I've renamed it and moved it to volatile state.

One question that did occur to me: can finished state be set from a snapshot eg. loading in a previous annotation and displaying it?

Reply:
It looks like the answer is no: https://mobx-state-tree.js.org/concepts/volatiles#modelvolatile

> Volatile props will not show up in snapshots, and cannot be updated by applying snapshots

Volatile state otherwise seems like a good pick for this use case. We _might_ be able to use `onSnapshot` to call an action that updates the volatile state though: https://mobx-state-tree.js.org/API/#onsnapshot

Reply:
Sorry, I meant should finished state be set from a snapshot in our rendering code? I vaguely remember there being a user story where we show you a finished annotation from someone else.

The transcription line stories have examples of calling `mark.finish()` to finish a mark. The only question would be, how does the rendering component know that a mark from an annotation snapshot is finished and cannot be edited at all.

Something to think about in the new year. Go and enjoy Christmas!

Reply:
Should we be able to load an incomplete classification from Panoptes and render those annotations as marks? I believe ASM allowed this since transcription of a document can take more than one session.

I think that's ok here because lines rendered from a saved annotation will always be finished.

Reply:
Yes, eventually. This should be in the user stories,.